### PR TITLE
Fix a broken link in contribution-checklists.md

### PR DESCRIPTION
**Pull request by chroju**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Output from acceptance testing:

```
N/A
...
```

### CODE REVIEW

File: docs/contributing/contribution-checklists.md [hunk 1]
```diff
@@ -626,8 +626,8 @@ guidelines.
    through long feedback cycles on a big PR with many resources. We ask you to
    only submit **1 resource at a time**.
 - [ ] __Acceptance tests__: New resources should include acceptance tests
-   covering their behavior. See [Writing Acceptance
-   Tests](#writing-acceptance-tests) below for a detailed guide on how to
+   covering their behavior. See [Writing An Acceptance
+   Test](running-and-writing-acceptance-tests.md#writing-an-acceptance-test) seciton for a detailed guide on how to
```
Comment (kamilturek) [inline docs/contributing/contribution-checklists.md:630]: There's a typo.

```suggestion
   Test](running-and-writing-acceptance-tests.md#writing-an-acceptance-test) section for a detailed guide on how to
```

**Review by kamilturek — CHANGES_REQUESTED** [review]

File: docs/contributing/contribution-checklists.md (hunk 1 above)
Comment (chroju) [reply docs/contributing/contribution-checklists.md]: Thanks for your review. I have fixed it.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

**Comment by ewbankkit** [comment]
@chroju Thanks for the contribution :tada: :clap:.

**Pull request merged**